packages/plugin-e2e/src/index.ts

```diff
@@ -1,4 +1,43 @@
+import { GrafanaPage, VariableEditPage } from './models';
+import { AlertPageOptions, AlertVariant, ContainTextOptions, PanelError } from './types';
+
 export { expect, test, type PluginFixture, type PluginOptions } from './api';
 export * from './e2e-selectors';
 export * from './fixtures';
 export * from './models';
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace PlaywrightTest {
+    const r: unique symbol;
+    const t: unique symbol;
+    interface Matchers<R, T> {
+      [r]: R;
+      [t]: T;
```

Comment:
This is the workaround for the problem described [here](https://github.com/microsoft/playwright/issues/12264#issuecomment-1797724200).